### DISCUSSION TITLE
There was an error encountered while trying to run chatGPT on Eclipse.

### DISCUSSION THREAD

**Discussion by Idrisiousionable** [community/community#57938] [Discussions] [answered]
### Select Topic Area

Question

### Body

Could someone assist me in resolving [this issue](https://github.com/gradusnikov/eclipse-chatgpt-plugin/blob/main/src/website/how-it-works-discuss.gif)? I have installed chatGPT on Eclipse using the guidelines provided in an article. However, when I attempted to execute Discuss on Assist AI, an error message was displayed.

Unable to run the task: java.lang.RuntimeException: java.io.IOException: Request failed with status code: 401 and response body: jdk.internal.net.http.ResponseSubscribers$HttpResponseInputStream@c432f2f java.lang.RuntimeException: java.io.IOException: Request failed with status code: 401 and response body: jdk.internal.net.http.ResponseSubscribers$HttpResponseInputStream@c432f2f

Please help me. Thank you all very much.

**Comment by queenofcorgis** [comment, marked as answer]
If you're asking how to use the piece of software that you found in a repository on GitHub, the best way to contact the maintainers of that software is to:

- Check the README for instructions on how to operate the software and pointers to documentation or troubleshooting info.
- Check the [SUPPORT file](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-support-resources-to-your-project), if one exists, for instructions on how to best contact the maintainers for support.
- Check the [CONTRIBUTING guide](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/setting-guidelines-for-repository-contributors), if one exists. Sometimes if there isn't a SUPPORT file, the CONTRIBUTING guide will give instructions on how to contact the maintainers for support.

All of these documents can be found, if they exist, in the repository where you found the software itself.

**Reply by Idrisiousionable** [reply]
Thank mate. I appreciate your assistance.

**Comment by datnguyen210394** [comment]
Great

**Comment by jge162** [comment]
> ### Select Topic Area
> Question
> 
> ### Body
> Could someone assist me in resolving [this issue](https://github.com/gradusnikov/eclipse-chatgpt-plugin/blob/main/src/website/how-it-works-discuss.gif)? I have installed chatGPT on Eclipse using the guidelines provided in an article. However, when I attempted to execute Discuss on Assist AI, an error message was displayed.
> 
> Unable to run the task: java.lang.RuntimeException: java.io.IOException: Request failed with status code: 401 and response body: jdk.internal.net.http.ResponseSubscribers$HttpResponseInputStream@c432f2f java.lang.RuntimeException: java.io.IOException: Request failed with status code: 401 and response body: jdk.internal.net.http.ResponseSubscribers$HttpResponseInputStream@c432f2f
> 
> Please help me. Thank you all very much.

Looks like your api key is wrong based on error message.

**Reply by Idrisiousionable** [reply]
According to your suggestions, I will implement the necessary corrections. Thank mate.
